README.md

```diff
@@ -20,39 +20,23 @@ Visual Studio Code version 1.37 or higher.
 
 ## Installation
 
-* Install from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=stoplight.spectral) 
+* Install from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=stoplight.spectral)
 * Install via the CLI: `code --install-extension stoplight.spectral`
-* Create a `.vsix` bundle using vsce (`npm install -g vsce`)  with `vsce package` and `code --install-extension {vsix filename}`
-
-## Development
-
-* Clone the [GitHub](https://github.com/stoplightio/vscode-spectral/) repository
-* Run `yarn`
-* Open the folder where you cloned the repository in VS Code
-* Start the included debugging configuration
-
-Please run `yarn run lint` and `yarn run test` before creating any pull-requests.
 
 ## Extension Settings
 
 This extension contributes the following settings:
 
-* `spectral.lintOnSaveTimeout`: Delay in ms before linting, default: `2000`
-* `spectral.defaultRuleset`: Default ruleset URI, default: `spectral:oas`
-* `spectral.clampRanges`: Minimize range highlighted by errors and warnings, default: `true`
-
-## Known Issues / Limitations
-
-If you open a single file in VS Code (as opposed to opening a folder or
-workspace), then [Spectral rulesets](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/rulesets.md) `(.?spectral.{yml|yaml|json}` files) will
-not be found, even if they are in the same directory. This is due to the use
-of the recommended/performant VS Code APIs for finding and watching files,
-which are exclusively workspace-based.
+* `spectral.enable`: Controls whether or not Spectral is enabled.
+* `spectral.rulesetFile`: Location of the ruleset file to use when validating. If omitted, the default is a `.spectral.yml`/`.spectral.json` in the same folder as the document being validated. Paths are relative to the workspace.
```

Comment:
```suggestion
* `spectral.rulesetFile`: Location of the ruleset file to use when validating. If omitted, the default is a `.spectral.yaml/.spectral.yml/.spectral.json/spectral.yaml/spectral.yml/spectral.json` in the same folder as the document being validated. Paths are relative to the workspace.
```
:laughing:

Reply:
Let's not mention the spectral.yaml but. I feel like that's a bug not a feature?

Reply:
@philsturgeon Huh? I'm sorry but I didn't get this. What rewording would you like to see?